game/scripts/npc/abilities/morphling_adaptive_strike_agi.txt

```diff
@@ -63,7 +63,7 @@
       "05"
       {
         "var_type"                                        "FIELD_FLOAT"
-        "shared_cooldown"                                 "3"
+        "shared_cooldown"                                 "1" //OAA
```

Comment:
This is interesting, but I still hate morphling. This change can become cancerous. Not really sure, because I can't be objective about this.